.eslintrc.json

```diff
@@ -38,7 +42,10 @@
       }
     ],
     "class-methods-use-this": "error",
-    "comma-dangle": "error",
+    "comma-dangle": [
+      "error",
+      "always-multiline"
```

Comment:
ESLint doc states: "...IE8 ... and below will throw an error when it encounters trailing commas in JavaScript."  Does MiQ have to support IE8?  If so, then I'm ok with these changes.  -thanks